README.md

```diff
@@ -151,3 +151,25 @@ In particular, note:
 * [targets factory methods](https://records-mover.readthedocs.io/en/latest/records_mover.records.targets.html)
 * [move() method](https://records-mover.readthedocs.io/en/latest/records_mover.records.html#records_mover.records.move)
 * [BaseRecordsFormat](https://records-mover.readthedocs.io/en/latest/records_mover.records.html#records_mover.records.base_records_format.BaseRecordsFormat)
+
+## Local Development
+
+The included Dockerfile can be used to build a docker image that is suitable for local development. Mount
+the directory containing your local copy of the repository to have it override what's in the container.
```

Comment:
Might want to add a line or two about doing docker build. Will need to be named `records-mover:latest` if I'm not wrong about your docker run command below. So

```bash
docker build -t records-mover:latest .
```

or similar